src/book/02-projects/04-other-projects/02-nupbr.mdx

```diff
@@ -0,0 +1,240 @@
+---
+section: Projects
+chapter: Other Projects
+title: NUpbr
+description: A physically-based rendering pipeline for generating realistic semi-synthetic data.
+slug: /projects/nupbr
+keywords:
+  [
+    vision,
+    data,
+    synthetic-data,
+    annotation,
+    deep-learning,
+    machine-learning,
+    learning,
+    ai,
+  ]
+---
+
+<Alert type='warning'>
+  NUpbr currently only supports Blender 2.79. Ensure this version is installed.
+</Alert>
+
+---
+
+## Motivation
+
+As the power of state-of-the-art deep learning methods increases, so does the requirement for large, fully annotated datasets. Particularly in terms of object recognition and classification computer vision tasks, annotation of this data requires considerable time and effort when performed manually by humans.
+
+The NUpbr project aims to reduce this annotation time by allowing for synthetic generation and automatic annotation of image data via the [Blender](https://www.blender.org/) 3D rendering engine. To ensure the data is comparable to that of realistically obtained data, [physically-Based Rendering (PBR)](https://en.wikipedia.org/wiki/Physically_based_rendering) materials are used. Additionally, [Image-Based Lighting (IBL)](http://ict.usc.edu/pubs/Image-Based%20Lighting.pdf) is used with High-Dynamic Range (HDR) environment maps to provide semi-synthetic image support.
+
+The tool is capable of outputting raw RGB images taken from random points of view within the scene, as well as generating the corresponding segmentation image, depth map, and various metadata for the current scene.
+
+---
```

Comment:
Maybe remove these breaks since we don't use them in other pages and it's good to keep consistency.